ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -62,40 +63,47 @@ type opennebula_im_mad_collectd = {
 
 type opennebula_im_mad_kvm = {
     include opennebula_im
-} = dict("arguments", '-r 3 -t 15 kvm');
+} = dict("arguments", '-r 3 -t 15 kvm', "sunstone_name", 'KVM');
 
 type opennebula_im_mad_xen = {
     include opennebula_im
-} = dict("arguments", '-r 3 -t 15 xen4');
+} = dict("arguments", '-r 3 -t 15 xen4', "sunstone_name", 'XEN');
 
 type opennebula_im_mad = {
     "collectd" : opennebula_im_mad_collectd
     "kvm" : opennebula_im_mad_kvm
-    "xen" : opennebula_im_mad_xen
+    "xen" ? opennebula_im_mad_xen
 } = dict();
 
 type opennebula_vm = {
     "executable" : string = 'one_vmm_exec'
     "arguments" : string
     "default" : string
+    "sunstone_name" : string
+    "imported_vms_actions" : string[] = list('terminate', 'terminate-hard', 'hold', 'release',
+                                             'suspend', 'resume', 'delete', 'reboot', 'reboot-hard',
+                                             'resched', 'unresched', 'disk-attach', 'disk-detach',
+                                             'nic-attach', 'nic-detach',
+                                             'snap-create', 'snap-delete')
```

Comment:
I would much rather see these each on their own line indented under the property.